src/lib/signature_lib/schnorr.ml

```diff
@@ -444,14 +498,18 @@ module Message = struct
 
   type t = (Field.t, bool) Random_oracle.Input.t [@@deriving sexp]
 
+  let network_id_mainnet = Char.of_int_exn 1
+
+  let network_id_testnet = Char.of_int_exn 0
```

Comment:
I think these constants should live in their own module, as I don't think only the signature lib will want to use them.